unleash-yggdrasil/Cargo.toml

```diff
@@ -20,9 +20,11 @@ convert_case = "0.6.0"
 unleash-types = "0.13.0"
 chrono = "0.4.38"
 dashmap = "5.5.0"
-hostname = "0.3.1"
 ipnetwork = "0.20.0"
 
+[target.'cfg(not(target_family = "wasm"))'.dependencies]
```

Comment:
This seemed more seamless than using features. WDYT?

Reply:
I think it's more seamless but less explicit. I kinda want the consumer to the have to opt into turning off the feature so they know what they're losing and not have their end user discover it. Otherwise it's kinda hidden and you don't know this will get sad at you until someone runs it or you know the code intimately